plasma_framework/test/src/exits/payment/PaymentChallengeIFENotCanonical.test.js

```diff
@@ -80,10 +89,26 @@ contract('PaymentInFlightExitRouter', ([_, ifeOwner, inputOwner, outputOwner, co
         };
     };
 
-    const buildValidNoncanonicalChallengeArgs = (decodedIfeTx, outputType) => {
-        const { inputTx } = createInputTransaction(outputType);
-
-        const { competingTx, decodedCompetingTx, competingTxPos } = createCompetitorTransaction(outputType);
+    const buildValidNoncanonicalChallengeArgs = (
+        decodedIfeTx,
+        outputType,
+        nonDepositInputTx = true,
```

Comment:
Minor: I would use `isDepositInputTx = false` seems a bit easier to read as a var.